images/pegasus/configs/dolphin/GCPadNew.ini

```diff
@@ -0,0 +1,120 @@
+[GCPad1]
+Device = evdev/0/Wolf X-Box One (virtual) pad
```

Comment:
This needs to be better fleshed out; Wolf next release will support hotplug and different types of joypads to better match the client configuration (Xbox/PS/Switch). Why do we need to hardcode the device type like that?

Reply:
Question about this: I was under the impression that wolf/moonlight just mapped everything to xbox controllers, is that not the case? If this isn't a good idea, would making a script that someone can run to setup different controllers be be better?